### ISSUE TITLE
Add cat /proc/cpuinfo to wrapper.sh?

### ISSUE THREAD

**Issue by klannon** [matz-e/lobster#594]
I'm trying to debug a non-Lobster batch issue and I'm finding I should have put `cat /proc/cpuinfo` into my script to help with the debugging.  I know we already dump a ton of random stuff into the task log when we run Lobster, but what do people think about adding the cpu info as well?

**Comment by matz-e** [comment]
Why not? Does it work with singularity? If yes, go for it!

**Comment by klannon** [comment]
OK, I'll make a to-do list item (for myself) to try this inside Singularity to make sure it's OK.  This means I need to learn how to launch jobs using singularity now that the front-ends are RHEL7.  So, it won't happen by tomorrow.  🙁

**Comment by annawoodard** [comment]
@klannon: I don't think you need anything special? From my bash history:
```
nohup /afs/crc.nd.edu/group/ccl/software/x86_64/redhat7/cctools/lobster-148-c1a7ecbd-cvmfs-0941e442/bin/work_queue_factory -T condor -M lobster_$USER.*ttV.*xsecs* -d all -o /tmp/${USER}_lobster_ttV_xsecs.debug -C $(readlink -f xsec_factory.json) --wrapper "python /afs/crc.nd.edu/group/ccl/software/runos/runos.py rhel6" --extra-options="--workdir=/disk" --worker-binary=/afs/crc.nd.edu/group/ccl/software/x86_64/redhat6/cctools/lobster-148-c1a7ecbd-cvmfs-0941e442/bin/work_queue_worker >& /tmp/${USER}_lobster_ttV_xsecs.log &
```

**Comment by klannon** [comment]
So, that's in your bash history (but not mine since I've never done it--and also don't use bash) but it's not in the Lobster documentation (at least not that I can find).  So, without that help, it would have been a long process for me to reconstruct.  Should we add some notes on Singularity to the Lobster docs too?

**Comment by annawoodard** [comment]
Sure thing @klannon. I created #595. First one to resolve it gets 15 internet points!

**Comment by annawoodard** [comment]
Double points for tenured professors

**Comment by klannon** [comment]
Fixed in 21d45ed.  I never remember to put that in the commit comment...

How many Internet points do I get for closing this and #595?
